shared/js/background/script-injection.js

```diff
@@ -0,0 +1,27 @@
+const browserWrapper = require('./wrapper.es6')
+
+// With Manifest V2 the content scripts are run directly from the manifest and
+// they inject into the main world of websites by adding a <script> element to
+// the DOM. With Manifest V3 however the chrome.scripting API must be used to
+// inject scripts into the main world.
+// Note: It's important that the isolated world script runs first. Script
+//       execution order is determined by the script ID.
```

Comment:
I'm not sure it is, https://source.chromium.org/chromium/chromium/src/+/main:components/on_load_script_injector/browser/on_load_script_injector_host.cc;l=46;drc=d00c1ed3052794f3bae53ad76b56209ecb290902 this code pushes it to the back of the queue if the ID doesn't match.

Reply:
Hmm, well I'm going from my testing. The order of the scripts in the `registerContentScripts` call made no difference, but the ID of the content script consistently determined which ran first. I have not looked at the Chromium code in detail though. I guess our integration tests will catch this if it changes.

Reply:
Cool happy to keep both strategies in case it got changed or will be etc :)

Reply:
I think it's since registerContentScripts puts the new script IDs into a std::set and then later iterates through those IDs to add the scripts. See https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/api/scripting/scripting_api.cc;l=929;drc=13dced01cce80364299ad7663479ad65069db73d;bpv=0;bpt=1. I will update the comment to say that the ordering is based on ID within the call.

Reply:
(Updated that comment.)